### PR TITLE
Update pl.po

### PR DESCRIPTION
**Pull request by adam23450**
Translations continued

### CODE REVIEW

File: src/assets/i18n/pl.po [hunk 1]
```diff
@@ -575,70 +575,70 @@ msgid ""
...
 
 msgid "AWS Region"
-msgstr "Region AWS"
+msgstr ""
 
 msgid "About"
 msgstr ""
 
 msgid "Access"
-msgstr "Dostęp"
```
Comment (erinix) [inline src/assets/i18n/pl.po]: I notice this PR removes some translations, was that intentional?

**Comment by adam23450** [comment]
It was not intended. It must have happened by mistake

File: src/assets/i18n/pl.po (hunk 1 above)
Comment (adam23450) [reply src/assets/i18n/pl.po]: From what I see everything is as it should be. Unless I look wrong? hmm

Comment (erinix) [reply src/assets/i18n/pl.po]: if you look at what it adds in in its place all you see is a "", I think your PR is somehow removing these, perhaps you were working off an older version of this file?

**Pull request closed without merging**